From b2fcbc0d9688e5602c7632aee1ec42ba7607ffa0 Mon Sep 17 00:00:00 2001
From: Tao Huang <huangtao@rock-chips.com>
Date: Sat, 12 Oct 2019 10:29:59 +0800
Subject: [PATCH] video: rockchip: rga2: fix clang warning

drivers/video/rockchip/rga2/rga2_mmu_info.c:284:20: warning: explicitly assigning value of variable of type 'uint32_t' (aka 'unsigned int') to itself [-Wself-assign]
            stride = stride;
            ~~~~~~ ^ ~~~~~~

Change-Id: I1c5d3b2f3354437777c8431ad027a98815f1fe70
Signed-off-by: Tao Huang <huangtao@rock-chips.com>
---
 drivers/video/rockchip/rga2/rga2_mmu_info.c | 1 -
 1 file changed, 1 deletion(-)

diff --git a/drivers/video/rockchip/rga2/rga2_mmu_info.c b/drivers/video/rockchip/rga2/rga2_mmu_info.c
index 51cf28ab49c9..f5d842684e0b 100644
--- a/drivers/video/rockchip/rga2/rga2_mmu_info.c
+++ b/drivers/video/rockchip/rga2/rga2_mmu_info.c
@@ -281,7 +281,6 @@ static int rga2_buf_size_cal(unsigned long yrgb_addr, unsigned long uv_addr, uns
         case RGA2_FORMAT_YCbCr_420_SP_10B:
         case RGA2_FORMAT_YCrCb_420_SP_10B:
             stride = (w + 3) & (~3);
-            stride = stride;
             size_yrgb = stride * h;
             size_uv = (stride * (h >> 1));
             start = MIN(yrgb_addr, uv_addr);
-- 
2.35.3

